Sitefinity CLI/Services/VisualStudioService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.Extensions.Logging;
+using Sitefinity_CLI.Exceptions;
+using Sitefinity_CLI.Model;
+using Sitefinity_CLI.PackageManagement;
+using Sitefinity_CLI.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+
+namespace Sitefinity_CLI.Services
+{
+    internal class VisualStudioService : IVisualStudioService
```

Comment:
We have VisualStudio worker that is encapsulating the VS related operations. (you use it here)
This class is not so related to visual studio. Also you have declared one method called InitializeSolution. 
In this method you are doing several stuffs. Maybe we should rename the class to VisualStuidoUpgradeService and the method should be. ExecuteVisualStudioUpgrade